server/utils/articles/article.util.js

```diff
@@ -57,14 +60,28 @@ class articleValidation {
       error.published.type = 'Publised must be a boolean';
     }
     if (!isEmptyObject(error.title)
-        || !isEmptyObject(error.body)
-        || !isEmptyObject(error.description)
-        || !isEmptyObject(error.tags)
-        || !isEmptyObject(error.published)) {
+      || !isEmptyObject(error.body)
+      || !isEmptyObject(error.description)
+      || !isEmptyObject(error.tags)
+      || !isEmptyObject(error.published)) {
       return Response(res, 422, error);
     }
     next();
   }
+
+  /**
+   * Checks if article exists
+   * @param {object} req
+   * @param {object} res
+   * @param {object} next
+   * @return {any} next/error
+   */
+  static async articleExist(req, res, next) {
+    const { articleId } = req.body;
+    const article = await Article.findByPk(articleId);
+    if (article === null) return Response(res, 400, 'Article does not exist');
```

Comment:
You may Kindly look into this error response too. I think it should be 404. what do you think?